### PR TITLE
Update for gui control parameters

### PR DESCRIPTION
**Pull request by linucks**
This allows us to display the pyrvapi gui even when running under jscofe or ccp4i2, and provides a mechanism for showing the gui under ccp4i

### CODE REVIEW

File: ample/util/argparse_util.py [hunk 1]
```diff
@@ -207,6 +204,9 @@ def add_general_options(parser=None):
     parser.add_argument('-scwrl_exe', metavar='path to scwrl',
                         help='Path to Scwrl4 executable')
 
+    parser.add_argument('-show_gui', metavar='True/False',
```
Comment (fsimkovic) [inline ample/util/argparse_util.py:207]: I much prefer this choice 👍

**Comment by fsimkovic** [comment]
Do your changes mean that AMPLE's GUI is no disabled by default and only pops up when invoked with `-show_gui True`?

**Comment by linucks** [comment]
Yes @fsimkovic - it only pops up when requested. I think this is better as it means that no gui appears when ample is run as a script, but also means that we can have our own gui pop up even when using ccp4i2.

**Comment by fsimkovic** [comment]
Awesome! A long-needed change ;-)

**Pull request merged**